client-admin/src/components/conversation-admin/conversation-has-comments-check.js

```diff
@@ -36,9 +36,10 @@ class ConversationHasCommentsCheck extends React.Component {
   }
 
   render() {
+    const { accepted_comments, rejected_comments, unmoderated_comments } = this.props;
     return (
       <div>
-        {this.props.accepted_comments > 0 && this.props.rejected_comments > 0 ? (
+        {accepted_comments !== null && rejected_comments !== null && unmoderated_comments !== null ? (
```

Comment:
if these properties have a value of `[]` then this statement will evaluate TRUE. Is that the intention? or would it be better to check if they are not NULL OR `[]` ?

Reply:
I thought I could do that before, but didn't understand redux reducers have a different sort of initial state, like here: 

https://github.com/pol-is/polisServer/blob/06987fb49362a7f267f780baf467929ff9d1216/client-admin/src/reducers/mod_comments_accepted.js#L5-L8

I should double-check, but I thought `null` state was importantly distance from empty array. Maybe the "loading" message  shown when state is null just isn't important though...

Reply:
Also, this is me reverted mostly back to what colin had before :)

Diff between this branch and the commit before I broke it:
https://github.com/pol-is/polisServer/compare/20ed3c69e91f3046119e77eadcd2b8b7b31a54c8%5E...396-comment-check-bug#diff-c62b7ef2074286d07bb49779fd4694c9